controls/listview/cells/listview-cells.md

```diff
@@ -6,64 +6,66 @@ position: 1
 slug: listview-cells
 ---
 # ListView Cell Types
-Cells are non visual objects used to describe a template for creating a visual element inside each ListView item. There are two types of cells:
```

Comment:
I have a hard time understanding the definition of a cell here. I looked at the definition in the MS docs, and it looks like they describe the same thing more effectively: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/listview/. Can we use a similar definition?

Reply:
Updated the description